include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -387,7 +387,8 @@ typedef struct OrtCUDAProviderOptions {
         has_user_compute_stream{},
         user_compute_stream{},
         default_memory_arena_cfg{},
-        tunable_op_enabled{false} {}
+        tunable_op_enabled{false},
```

Comment:
Maybe rename the flags to `tunable_op_use_enabled` and `tunable_op_tuning_enabled` to be more descriptive?

Reply:
https://github.com/microsoft/onnxruntime/pull/15156#issuecomment-1493719115